src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -410,6 +412,24 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         auto pfnScrollPositionChanged = std::bind(&TermControl::_TerminalScrollPositionChanged, this, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);
         _terminal->SetScrollPositionChangedCallback(pfnScrollPositionChanged);
 
+        // Set up blinking cursor
+        int blinkTime = GetCaretBlinkTime();
+        if (blinkTime != INFINITE)
+        {
+            // Create a timer
+            _cursorTimer = std::make_optional(DispatcherTimer());
+            _cursorTimer.value().Interval(std::chrono::milliseconds(blinkTime));
+            auto registrationtoken = _cursorTimer.value().Tick({ this, &TermControl::_BlinkCursor });

```

Comment:
Looks like `registrationtoken` is unused after here, so I think you can go ahead and skip the assignment, and leave it as 
```
_cursorTimer.value().Tick({ this, &TermControl::_BlinkCursor });
```